tests-beta/ios/MASVS-CODE/MASVS-TEST-0x80-3.md

```diff
@@ -0,0 +1,26 @@
+---
+Title: Testing Enforced Updating
+ID: MASTG-TEST-0x80
+Link: https://mas.owasp.org/MASTG/tests/android/MASVS-CODE/MASTG-TEST-0080/
+Platform: ios
+MASVS v1: ['MSTG-ARCH-9']
+MASVS v2: ['MASVS-CODE-2']
+type: [static]
+---
+
+## Overview
+
+When a vulnerability is found in the app, it should be possible to force the user to update the application to continue using it.
+
+## Steps
+
+1. Examine the startup flow of the application. Identify if the application calls out to a backend with the application's version information included.
+2. Examine if the application can handle a specific response from the backend indicating that the application must be updated. For example, the application might evaluate the response from the backend and show a specific error message. Note that the error message can also come from the backend, so the lack of a custom error message in the application does not mean that the feature isn't implemented.
```

Comment:
Same as above. Regardless of my comment, the note you are using is a bit confusing to me. Otherwise, I like very much the rest.